modules/migration-known-issues.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// migration/migrating_openshift_3_to_4/migrating-openshift-3-to-4.adoc
+[id='migration-known-issues_{context}']
+= Known issues
+
+The following issues will be addressed in a future release:
```

Comment:
I'd be careful about saying this, these issues might not ever get fixed.

Reply:
What do you suggest I use as a lead-in? "The following are known issues:"?

Reply:
You could do anything like:

* This release has the following known issues:
* Review the following known issues: